.github/workflows/oppiabot.yml

```diff
@@ -4,13 +4,22 @@ on:
   issues:
     types:
       - labeled
+  pull_request_target:
+    branches:
+      - develop
+      - release-*
 
 jobs:
   oppiabot:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Ensure that appropriate labels are added to issues
-        uses: oppia/oppiabot@1.1.0
+      - name: Github Actions from Oppiabot
+        uses: oppia/oppiabot@1.2.0
         with:
           repo-token: ${{secrets.GITHUB_TOKEN}}
+        env:
+          SHEETS_TOKEN: ${{ secrets.SHEETS_TOKEN }}
+          SHEETS_CRED: ${{ secrets.SHEETS_CRED }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}`
```

Comment:
We are already providing this value with repo-token, do we still need to set the same value on env?

Reply:
The script for the CLA Check requires the GITHUB_TOKEN to be an environment variable.

Reply:
> I'll make an issue in Oppiabot to remove either one of them in a later release.

Is it done?

Reply:
The release was made with the changes. DONE